scripts/exp/tools/README.md

```diff
@@ -0,0 +1,148 @@
+### Tools Directory
+
+This directory provides scripts to run experiments with different environment configurations, collect results, and demonstrate usage through an example script.
+
+### Directory Structure
+
+scripts/exp/tools/
+├── run_envs.sh       # Script for running experiments
+├── collect.py        # Results collection and summary
+├── test_system.sh    # Usage script for rdagent kaggle loop
+├── README.md         # This documentation
+
+Tools Overview
+
+	1.	run_envs.sh: Executes experiments with different environment configurations in parallel.
+	2.	collect.py: Collects and summarizes experiment results into a single file.
+	3.	test_system.sh: Demonstrates how to use the above tools together for experiment execution and result collection [for rdagent kaggle loop]
+
+### Getting Started
+
+Prerequisites
+
+1. Ensure the scripts have execution permissions:
+
+```
+chmod +x scripts/exp/tools/run_envs.sh
```

Comment:
You can commit the file permission to github directly instead of writting them in README.md